src/main/java/yapyap/Task.java

```diff
@@ -0,0 +1,72 @@
+package yapyap;
+
+/**
+ * Represents a task with a description and a completion status.
+ * This class serves as a base class for more specific types of tasks such as Todos, Deadlines, and Events.
+ */
+public class Task {
```

Comment:
Perhaps a separate Todo class to handle Todo tasks, while Task is an abstract class?